test/runnable/future.d

```diff
@@ -0,0 +1,43 @@
+/* PERMUTE_ARGS:
```

Comment:
Since there is code in `toobj.d` to support DIP1007, I feel like running it is a more foolproof approach. Is there a reason why it shouldn't be run ?